src/github/types/webhook-events.ts

```diff
@@ -1,2 +1,23 @@
-import { emitterEventNames } from "@octokit/webhooks/dist-types/generated/webhook-names.js";
+import { emitterEventNames } from "@octokit/webhooks";
```

Comment:
```suggestion
import { emitterEventNames, EmitterWebhookEventName as GitHubEventClassName } from "@octokit/webhooks";
```
you can just import the union type